rainforest-cli.go

```diff
@@ -152,7 +152,7 @@ func main() {
 			Aliases:      []string{"r"},
 			Usage:        "Run your tests on Rainforest",
 			OnUsageError: onCommandUsageErrorHandler("run"),
-			Action:       func(c *cli.Context) error {
+			Action: func(c *cli.Context) error {
```

Comment:
Is it right that some are "line up" formatted and some are not? `func` is an exception? That would be understandable though.

Reply:
It was autofixed and the check isn't complaining, so that's probably the convention.